subsys/bluetooth/controller/hci_driver.c

```diff
@@ -31,7 +31,21 @@ static K_THREAD_STACK_DEFINE(recv_thread_stack, CONFIG_BLECTLR_RX_STACK_SIZE);
 static K_THREAD_STACK_DEFINE(signal_thread_stack,
 			     CONFIG_BLECTLR_SIGNAL_STACK_SIZE);
 
-static u8_t ble_controller_mempool[CONFIG_BLECTRL_MEMPOOL_SIZE];
+#define MASTER_MEM_SIZE (BLE_CONTROLLER_MEM_PER_MASTER_LINK( \
+	CONFIG_BT_CTLR_DATA_LENGTH_MAX, \
```

Comment:
`CONFIG_BT_CTLR_DATA_LENGTH_MAX` is unavailable if `BT_DATA_LEN_UPDATE` is disabled